README.md

```diff
@@ -6,29 +6,42 @@ The reason for this at times perhaps slightly inconvenient interface is it makes
 
 ```OCaml
 module Make(B : Mirage_block.S)(Clock : Mirage_clock.PCLOCK) = struct
-
+  (* Create a module for partitioning a Mirage block device *)
   module Partitioned = Mirage_block_partition.Make(B)
+
+  (* Create a module for accessing a tar KV store on a partitioned block device *)
   module Tar = Tar_mirage.Make_KV_RO(Partitioned)
-  module Chamelon = Kv.Make(Partitioned)(Clock)
+
+  (* Create a module for accessing a Chameleon filesystem on a partitioned block device *)
+  module Chameleon = Kv.Make(Partitioned)(Clock)
+
+  (* Connect to the block device and partition it into three sub-blocks *)
+  let%bind connect_and_partition b =
+    let%bind b1, rest = Partitioned.connect (Sectors.of_int 20) b in
+    let b2, b3 = Partitioned.subpartition (Sectors.of_int 8192) rest in
+    (* Return a tuple containing the three sub-blocks *)
+    return (b1, b2, b3)
 
   let start b =
     let open Lwt.Syntax in
-    (* b1 is the first twenty sectors, b2 is the next 8k sectors (4 MiB),
-       and b3 is the remaining space. Note that the initial [connect] call is
-       asynchronous while the later [subpartition] calls are not. If the
-       partition point is outside the block device or subpartition then an
-       exception is raised. *)
-    let* b1, rest = Partitioned.connect 20L b in
-    let b2, b3 = Partitioned.subpartition 8192L rest in
-    (* now use e.g. b1 as a tar KV store, b2 as a chamelon filesystem,
-       b3 as a raw block device... *)
-    let* tar = Tar.connect b1
-    and* chamelon = Chamelon.connect ~program_size:16 b2 in
-    ...
+    (* Connect to the block device and partition it into three sub-blocks *)
+    let* b1, rest = Partitioned.connect (Sectors.of_int 20) b in
```

Comment:
The function Sectors.of_int is utilized to transform an integer value into a Sectors.t value, which is a type alias that denotes an Int64.t value that signifies the quantity of disk sectors measuring 512 bytes. The Mirage_block library or a library that depends on it may contain the Sectors module, which is most likely defined elsewhere in the codebase. It's difficult to pinpoint exactly where Sectors and Sectors.of_int originate without additional context or details, but they are probably defined somewhere in the used codebase.

Reply:
What does "most likely defined elsewhere" mean? And what does the function do with the int?